src/test/java/com/eleks/academy/whoami/repository/GameInMemoryRepositoryTest.java

```diff
@@ -19,13 +20,23 @@ public class GameInMemoryRepositoryTest {
 
 	private final NewGameRequest gameRequest = new NewGameRequest();
 
-	private final GameInMemoryRepository gameRepository = new GameInMemoryRepository();
```

Comment:
better write it like this:
```
private GameInMemoryRepository gameRepository;

@BeforeEach
	public void setup() {
                gameRepository = new GameInMemoryRepository();
		gameRequest.setMaxPlayers(4);
	}
```
No need to use `@InjectMocks` when it doesn't have any dependencies